src/%IPM/Repo/Remote/Definition.cls

```diff
@@ -27,7 +27,6 @@ XData Commands [ XMLNamespace = "http://www.intersystems.com/PackageManager/CLI"
 {
 <commands>
 <command name="repo">
-<group name="Remote repository">
```

Comment:
Why is the group name deleted?

Reply:
@isc-kiyer  Actually, I think we should remove the XDATA blocks in this class (Remote.Definition.cls), %IPM.Repo.Extension.Perforce.Definition.cls, and %IPM.Repo.Filesystem.Definition.cls. This is because all of those CLI commands are actually defined in Main.cls so those are duplications. (Our current HSPM also defines everything all together at PackageManager.cls).
Also, I removed the <group> tag because by the change ported from HSPM to %IPM.CLI.Command.cls, <xs:element name="group"> has been removed (https://github.com/Beiming-Zhang/zpm/pull/12/files#:~:text=%3Cxs%3Aelement%20name%3D%22group%22%3E) but not added back in the new xs:schema

Reply:
@Beiming-Zhang No we should keep them and remove them from Main. This is a nice system where we can delegate adding new modifiers to existing commands only when applicable. For example, none of the perforce modifiers should be available for the main repo command. Should only be available when the zpm-perforce module is also installed.

Reply:
Have modified the Main.cls's XDATA block accordingly & keep the ones in separate classes